MantidPlot/src/ApplicationWindow.cpp

```diff
@@ -13625,47 +13625,47 @@ void ApplicationWindow::translateActionsStrings() {
   conestyle->setToolTip(tr("Cones"));
   conestyle->setStatusTip(tr("Cones"));
   crossHairStyle->setText(tr("Crosshairs"));
-  crossHairStyle->setMenuText(tr("Crosshairs"));
+  crossHairStyle->setText(tr("Crosshairs"));
```

Comment:
True. Then this is totally redundant now. I wonder what the original code was meaning to do here. I'll just remove those lines, as the only replacement for `setMenuText()` is `setText()`.